lib/manager/maven/extract.ts

```diff
@@ -310,18 +365,43 @@ export async function extractAllPackageFiles(
   packageFiles: string[]
 ): Promise<PackageFile[]> {
   const packages: PackageFile[] = [];
+  const additionalRegistryUrls = [];
+
   for (const packageFile of packageFiles) {
     const content = await readLocalFile(packageFile, 'utf8');
     if (content) {
-      const pkg = extractPackage(content, packageFile);
-      if (pkg) {
-        packages.push(pkg);
+      if (packageFile.endsWith('settings.xml')) {
+        const registries = extractRegistries(content);
+        if (registries) {
+          logger.debug(
+            { registries, packageFile },
+            'Found registryUrls in settings.xml'
+          );
+          additionalRegistryUrls.push(...registries);
+        }
       } else {
-        logger.debug({ packageFile }, 'can not read dependencies');
+        const pkg = extractPackage(content, packageFile);
+        if (pkg) {
+          packages.push(pkg);
+        } else {
+          logger.debug({ packageFile }, 'can not read dependencies');
```

Comment:
Should we lower this to trace?
```suggestion
          logger.trace({ packageFile }, 'can not read dependencies');
```